src/reformatters/ecmwf/ifs_ens/forecast_15_day_0_25_degree/template_config.py

```diff
@@ -92,11 +91,10 @@ def dimension_coordinates(self) -> dict[str, Any]:
                     pd.timedelta_range("150h", "361h", freq="6h")
                 )
             ),
-            "ensemble_member": np.arange(
-                0, 51
-            ),  # single control member (0) + 50 perturbed members (1-50)
+            # single control member (0) + 50 perturbed members (1-50)
+            "ensemble_member": np.arange(0, 51),
             "latitude": np.flip(np.arange(-90, 90.25, 0.25)),
-            "longitude": np.arange(-180, 180, 0.25),
+            "longitude": np.arange(-180, 179.75, 0.25),
```

Comment:
No longer convinced this is correct for all the gribs. Looking more closely. I'm getting shape errors when backfilling more than 2 days.